plugin.xml

```diff
@@ -25,27 +25,26 @@
  -->
 
     <platform name="android">
-        <config-file target="res/xml/config.xml" parent="/*">
+        <preference name="ANDROID_SUPPORT_V4_VERSION" default="27.+"/>
+        <config-file target="config.xml" parent="/*">
             <feature name="SitewaertsDocumentViewer">
                 <param name="android-package"
                        value="de.sitewaerts.cordova.documentviewer.DocumentViewerPlugin"/>
             </feature>
         </config-file>
 
-        <config-file target="AndroidManifest.xml" parent="/manifest">
-            <uses-permission
-                    android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+        <config-file target="AndroidManifest.xml" parent="/*">
+            <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
         </config-file>
 
         <!--see http://stackoverflow.com/questions/38200282/android-os-fileuriexposedexception-file-storage-emulated-0-test-txt-exposed-->
         <!--see https://github.com/sitewaerts/cordova-plugin-document-viewer/issues/105-->
         <framework src="com.android.support:support-v4:27+" />
 
-        <config-file target="AndroidManifest.xml" parent="/manifest/application">
-
-             <provider
+        <config-file target="AndroidManifest.xml" parent="application">
+            <provider
                 android:name="de.sitewaerts.cordova.documentviewer.FileProvider"
-                android:authorities="$PACKAGE_NAME.DocumentViewerPlugin.fileprovider"
+                android:authorities="${applicationId}.DocumentViewerPlugin.fileprovider"
```

Comment:
Why did you change this from $PACKAGE_NAME to ${applicationId}? Is that change relevant to fix the issue?

Reply:
@regnete  This will be help full if you have multiple variants(dev, staging prod) for your app and you want to install all the builds simultaneously. Then applicationId will help or else it will throw the following error
`Failure [INSTALL_FAILED_CONFLICTING_PROVIDER: Scanning Failed.:`